pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md

```diff
@@ -9,41 +9,49 @@ contributors:
+* [AMP Tools](../../../documentation/tools.html?format=websites) 
 
 
-*   [Validate AMP pages](../../../documentation/guides-and-tutorials/learn/validation-workflow/validate_amp.md?format=websites)
-*   [The AMP Validator ](https://validator.ampproject.org/)
-*   [Google AMP Tester](https://search.google.com/test/amp)
-*   [AMP Tools](../../../documentation/tools.html?format=websites) 
+# Grant cached AMP pages server access
 
+Great news, valid AMP pages automatically opt into all existing AMP Caches! This means your users experience content that loads efficiently and safely. These types of optimizations are great, but come with a small catch.
+Some users will be served AMP pages from domains that do not match your own. This can cause pages to lose access to site data when using dynamic AMP components such as [`<amp-form>`](../../../documentation/components/reference/amp-form.md?format=websites) or [`<amp-list>`](../../../documentation/components/reference/amp-list.md?format=websites). These types of errors are Cross-Origin Resource Sharing, or CORS, issues. Work with safety, not against it, by enabling CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json)! 
 
-# Grant cached AMP documents server access
+Learn more about accessing server data:
 
-Valid AMP pages are automatically opted into all existing AMP caches. This is great news! Your pages are automatically preloaded efficiently and safely, enabling user-beneficial performance optimizations to content. However, documents served from AMP caches are served from domains that do not match your own. This can cause  Cross-Origin Resource Sharing (CORS) issues when using dynamic AMP components, such as [`<amp-form>`](../../../documentation/components/reference/amp-form.md?format=websites) or [`<amp-list>`](../../../documentation/components/reference/amp-list.md?format=websites), causing your site to lose access to your data. This means certain content may not load correctly or at all.  It's important to enable CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json) for your pages to work as intended for all possible users. 
+* [How AMP Pages are cached ](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_amp_pages_are_cached.md?format=websites)
+* [CORS in AMP](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.md?format=websites)
+* [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
 
-Learn more about accessing server data:
 
+# Shareable, safe, and substantiated content with signed exchanges 
 
+Keep your domain's URL while maintaining portable content by implementing signed exchanges (SXG). By serving AMP pages with SXG, digital signatures protect your information by tying the document to its claimed URL. Implementing SXG delivers signed AMP content in addition to, rather than instead of, regular AMP content.
```

Comment:
what does `maintaining portable content ` mean? Maybe mention the benefit of easier analytics and user sessions as cookies are always first party.